crates/nargo_cli/Cargo.toml

```diff
@@ -37,7 +37,7 @@ termcolor = "1.1.2"
 color-eyre = "0.6.2"
 
 # Backends
-acvm-backend-barretenberg = { git = "https://github.com/noir-lang/aztec_backend", rev = "c9fb9e806f1400a2ff7594a0669bec56025220bb", default-features=false }
```

Comment:
This has been merged and needs to be changed to `677f10e07011849f8aa0d75fe80390bb3081b1e5`

Reply:
Done